src/pages/Privacy/PopUp.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Link from '../../components/common/Link';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import { CardHeader } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '360px',
+    height: '206px',
+    zIndex: 50000,
+    position: 'fixed',
+    right: '1.51%',
+    bottom: '0.5%',
+  },
+  title: {
+    fontSize: '20px',
+    fontWeight: '700',
+  },
+  headStyle: {
+    padding: 8,
+  },
+  iconStyle: {
+    verticalAlign: 'bottom',
+  },
+  copyStyle: {
+    paddingTop: 5,
+    width: '308px',
+  },
+  hoverContent: {
+    fontSize: 12,
+    textAlign: 'justify',
+    lineHeight: '16.2px',
+  },
+  linkStyle: {
+    padding: 0,
+    color: '#FFB100',
+    marginLeft: 3,
+    textDecoration: 'none',
+  },
+  imageStyle: {
+    width: '26px',
+    height: '26px',
+    marginRight: '2%',
+    verticalAlign: 'middle',
+  },
+  buttonStyle: {
+    width: '150px',
+    height: '36px',
+    color: '#0F1D2F',
+    fontWeight: '700',
+  },
+  linkText: {
+    color: '#0F1D2F',
+    fontWeight: '700',
+    paddingRight: '25px',
+  },
+}));
+
+const PopUp = () => {
+  const classes = useStyles();
+  const date = new Date();
+  const setCookie = function (cookieEmail, cookieValue, expDays) {
+    date.setTime(date.getTime() + expDays);
+
+    const expires = 'expires=' + date.toUTCString();
+
+    document.cookie = cookieEmail + cookieValue + ';' + expires + ';path=/';
+
+    return document.cookie;
+  };
+
+  const deleteCookie = function (cookieEmail) {
+    date.setTime(date.getTime());
+    const expires = 'expires=' + date.toUTCString();
+    document.cookie = cookieEmail + expires + ';path=/';
+  };
+  const acceptCookieConsent = () => {
+    deleteCookie('civictecindex_cookie_consent');
+    setCookie('civictecindex_cookie_consent', 1, 30);
```

Comment:
Add missing letter 'h' to 'civictechindex'
```suggestion
    deleteCookie('civictechindex_cookie_consent');
    setCookie('civictechindex_cookie_consent', 1, 30);
```